locale/en/LC_MESSAGES/django.po

```diff
@@ -2078,7 +2078,7 @@ msgid ""
 "basis van een eerder uitgevoerde studie. Indien je wijzigingen hebt in een "
 "al goedgekeurde studie die daarom opnieuw getoetst moet worden, maak dan <a "
 "href=\"%(amendment_url)s\">hier</a> een amendement aan."
-msgstr "This page is only intended if you are going to do a completely new study based on a previously conducted study. If you have changes to an already approved study that therefore needs to be re-reviewed, please make an amendment <a href=\"%(amendment_url)s\">here</a>."
+msgstr "This page is only intended for creating an entirely new study based on a previously conducted study. If you have changes to an already approved study that therefore needs to be re-reviewed, please make an amendment <a href=\"%(amendment_url)s\">here</a>."
```

Comment:
This is what I had before (as it's much better), but Desiree changed it to the wording eventually used here

Reply:
Maybe change it to "if you are going to start a new study"?

Because I think Desiree changed it because she wanted to emphasize starting a new study in terms of a new research study